docs/fio_distributed.md

```diff
@@ -0,0 +1,28 @@
+# FIO Distributed
+
+FIO (Flexible I/O Tester) has a native mechanism to run multiple servers concurrently and summarize
```

Comment:
Would be better if we can add a bit more documentation on how there's a client that gets launched, and the interaction of the client pod with the server pods if possible. I think this would be useful for a newcomer.

Reply:
I added a single line, but that interaction is documented in the fio client pretty well. I would advise people to look there.